src/libsyntax/config.rs

```diff
@@ -161,7 +161,8 @@ impl<'a> StripUnconfigured<'a> {
                                  "stmt_expr_attributes",
                                  attr.span,
                                  GateIssue::Language,
-                                 EXPLAIN_STMT_ATTR_SYNTAX);
+                                 EXPLAIN_STMT_ATTR_SYNTAX,
+                                 Some("If you just want to comment your code, just use '//'"));
```

Comment:
Won't this `help` trigger for all misplaced attributes? It should only trigger for misplaced `doc` attributes.